test/python/WMCore_t/WorkQueue_t/Policy_t/Start_t/MonteCarlo_t.py

```diff
@@ -88,7 +88,7 @@ def testLHEProductionWorkload(self):
             SliceSize = LHEProductionWorkload.startPolicyParameters()['SliceSize']
             self.assertEqual(math.ceil(float(totalevents) / (SliceSize * splitArgs['MaxJobsPerElement'])),
                              len(units))
-            first_event = 1
+            first_event = 1.0
```

Comment:
That is really weird. In any case, it seems this is a good time to 

```
from __future__ import division
``` 

for this test rather than mucking around with promoting things to get python 2 to do the right thing. Can you try that instead?